### PR TITLE
Tricot traits

### PR DESCRIPTION
**Pull request by marieALaporte**
Description 
-----------------------------------------------------
Added the Tricot variables needed for ClimMob trials integrations. 
Would be great if you can load those on the test server as well

Checklist 
---------------------------------------------------------------------------------------
- [ ] An OBO file generated from an OWL file
- [ ] New terms
  - [ ] The relevant issue has been closed.
  - [ ] Further work is required.
- [ ] Term updates
  - [ ] The relevant issue has been closed.
  - [ ] Further work is required.
- [ ] CHECKS
  - [ ] New variables have a parent trait ID 
    - [ ] New variables have methods and scales 
  - [ ] The OBO file has been validated
    - [ ] checked in cassavabase
    - [ ] checked CropOntology

### CODE REVIEW

**Comment by marieALaporte** [comment]
@qlands

File: cassava_trait.obo [hunk 1]
```diff
@@ -13908,6 +14784,13 @@ namespace: CassavaScale
 is_a: CO_334:0001001 ! Scales
 relationship: scale_of CO_334:0013051 ! white color appearance method
 
+[Term]
+id: CO_334:0103077
+name: Block:3
```
Comment (nmenda) [inline cassava_trait.obo:14789]: what is 'block 3'? 
Do we want to use 'tricot block 3' for all the new variables? Are those tricot specific or should those be allowed to be reused by other projects/breeders?

File: cassava_trait.obo [hunk 2]
```diff
@@ -8770,6 +8769,876 @@ relationship: variable_of CO_334:0013012 ! attieke-like aroma method
 relationship: variable_of CO_334:0103029 ! Attieke aroma scale 0-10
 created_by: Ebah Catherine
 
+[Term]
+id: CO_334:0003144
+name: Canopy formation Tricot block 3
```
Comment (nmenda) [inline cassava_trait.obo:8774]: should 'Tricot block 3' be part of this variable? can it be made generic so others can reuse it?

File: cassava_trait.obo (hunk 1 above)
Comment (marieALaporte) [reply cassava_trait.obo:14789]: The tricot variables have all the same method and scale. This block 3 means that for each variable you get an array of 3 varieties (the best of the three , the one in the middle and the worse of the 3).

File: cassava_trait.obo (hunk 2 above)
Comment (marieALaporte) [reply cassava_trait.obo:8774]: I guess you can still reuse the trait to create a variable that's not climmob specific.

Comment (nmenda) [reply cassava_trait.obo:8774]: is there a way then to remove the Tricot language and 'block 3' ? I think it should be possible to define how traits like 'canopy formation' are assayed without specifying ''block 3". If someone else uses an array of 4 or 5 they should still use the same variable.

File: cassava_trait.obo [hunk 3]
```diff
@@ -8770,6 +8769,876 @@ relationship: variable_of CO_334:0013012 ! attieke-like aroma method
...
+
+[Term]
+id: CO_334:0003145
+name: Discoloration Tricot block 3
+namespace: cassava_trait
+synonym: "Discol_CompRank_Bl3" EXACT []
+is_a: CO_334:0001000 ! Variables
+relationship: variable_of CO_334:0003204 ! Discoloration
+relationship: variable_of CO_334:0013054 ! Comparative ranking by users method
+relationship: variable_of CO_334:0103077 ! Block:3
```
Comment (nmenda) [inline cassava_trait.obo:8790]: all the variables with 'block 3' scale should have the actual scale used for measuring each trait. Whether it's a quantitative unit or a nominal scale, etc. 

I also suggest to completely omit 'block 3' from the variables to make them reusable by others.

File: cassava_trait.obo (hunk 2 above)
Comment (elizabetharnaud) [reply cassava_trait.obo:8812]: In fact, we agreed while ago with Jacob  to not use the term 'TRICOT' but use  'comparative ranking method' instead with a  block number that indicates the number of varieties compared. So, " block:3" is an important information as it the number of technologies compared is part of the method used so it is part of the variable.

**Pull request closed without merging**